.github/workflows/tests.yml

```diff
@@ -172,9 +172,9 @@ jobs:
         run: sudo apt install -y libgpgme-dev libbtrfs-dev libdevmapper-dev
 
       - name: Run golangci-lint
-        uses: golangci/golangci-lint-action@v6
+        uses: golangci/golangci-lint-action@v7
         with:
-          version: v1.61.0
+          version: latest
```

Comment:
Let's not do this.  This makes it unpredictable when the linter version will change, which means we can get unrelated linting errors on PRs and not know why.  It also forces unrelated changes into PRs.  We could be making a small docs PR, get a new `latest` linter version, and suddenly have to start changing code to get it through.

Reply:
Well, I do not see occasional linter error which can be solved by bumping your local linter to the newest possible version as such a drag compared to the problem I am trying to solve, but sure. If you don't like this then I believe this PR is a dead end. Closing.

Reply:
What about updating the action to v2 though?  I think that's certainly worth doing, I just don't like the idea of a "floating" version for the linter itself.

Reply:
Sure I can do that, but in a different PR.